app/controllers/search_controller.rb

```diff
@@ -1,12 +1,14 @@
 class SearchController < ApplicationController
   def index
     return if params[:query].nil?
-    return if params[:query].empty?
 
-    @exact_results = add_results_for(params[:query])
-    @more_results = more_results(params, @exact_results)
+    unless params[:query] == ""
+      @exact_results = add_results_for(params[:query])
+      @more_results = more_results(params, @exact_results)
```

Comment:
It's a bit weird that two search functions are named differently and have different signatures (one takes a search string, the other all params). If you have some spare time, this could be refactored, but I know you don't... :D

Reply:
done